rust/Cargo.toml

```diff
@@ -66,7 +66,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_derive = { version = "1.0" }
 serde_json = { version = "1.0" }
 serde_urlencoded = "0.7.1"
-sqlx = { version = "0.7", features = [
+sqlx = { version = "0.8.2", features = [
```

Comment:
Had to bump due to a dependency resolution failure, but got a nice thing out of it (no longer have to manually implement a certain type mapping in cyclotron)